### ISSUE TITLE
How to support geotiff?

### ISSUE THREAD

**Issue by sandorkertesz** [ecmwf/earthkit-data#120]
This issue was created to discuss the possible support for geotiff  in earthkit-data.

**Comment by milanavuckovic** [comment]
Thanks for opening this issue, Sandor.
Support for geotiff data would be much appreciated in CLIMAAX project.

**Comment by sandorkertesz** [comment]
I tried to use https://github.com/KipCrossing/geotiff to read geotiff, a `noGdal` project, but I immediately hit a limitation with the geotiffs from the CLIMAAX Land use data:

```
  255     return temp_crs_code
    256 elif temp_crs_code == 32767:
--> 257     raise UserDefinedGeoKeyError(crs_error_msg)
    258 else:
    259     raise GeographicTypeGeoKeyError()

UserDefinedGeoKeyError: User-defined GeoKeys are not yet supported.
```

The githiub page actually mention it as a core feature yet to be implemented:
"read a user defined CRS 32767 from tiff file"

So I am not sure we can use it in earthkit-data.

**Comment by corentincarton** [comment]
Tagging @chpolste, who will investigate this for the CLIMAAX project.

**Comment by corentincarton** [comment]
From @colonesej, this link could be interesting: https://kipling.medium.com/nogdal-e5b60b114a1c

**Comment by sandorkertesz** [comment]
Fixed by #503

**Comment by milanavuckovic** [comment]
[celebrate]     Milana Vuckovic reacted to your message:
________________________________
